src/GoogleSignin.js

```diff
@@ -9,12 +9,14 @@ const IS_IOS = Platform.OS === 'ios';
 export const statusCodes = {
   SIGN_IN_CANCELLED: RNGoogleSignin.SIGN_IN_CANCELLED,
   IN_PROGRESS: RNGoogleSignin.IN_PROGRESS,
+  PLAY_SERVICES_NOT_AVAILABLE: RNGoogleSignin.PLAY_SERVICES_NOT_AVAILABLE,
 };
 
 class GoogleSignin {
   configPromise;
 
   async signIn() {
+    await this.hasPlayServices();
```

Comment:
I agree with you that calling `hasPlayServices` for the users is not ideal for a couple of reasons but most importantly it's totally unnecessary in most of the cases. If the user is concerned about availability of Play Services they should make that choice themselves and hide Google Sign In button accordingly.

I would remove `hasPlayServices` check from `signIn` method and explain in the docs why calling `hasPlayServices` might be useful. For example hiding google sign in button and providing alternative sign in options.

Reply:
good points! I'll fix this today and get back to you.

Reply:
> it's totally unnecessary in most of the cases

actually I just recently tried an app that uses this package on an android 8.1 tablet and you'd be surprised - I got the modal alert saying I need to update play services! I think it may be because we're using the latest version of the dependencies and the tablet needed to update play services first.